src/main/kotlin/racingcar/Cars.kt

```diff
@@ -0,0 +1,48 @@
+package racingcar
+
+import java.util.stream.IntStream
+import kotlin.streams.toList
+
+class Cars(private val cars: List<Car>) {
+    fun forwardAllByCondition(conditions: List<Int>) {
+        cars.mapIndexed { index, car ->
+            forwardByCondition(car, conditions[index])
+        }
+    }
+
+    fun forwardByCondition(car: Car, condition: Int) {
+        if (canForward(condition)) {
+            car.forward()
+        }
+    }
+
+    private fun canForward(condition: Int) = condition >= MAX_FORWARD_CONDITION
+
+    fun getSize(): Int {
+        return cars.size
+    }
+
+    fun getPositionByIndex(index: Int): Int {
+        return cars[index].position
+    }
+
+    fun getPositions(): List<Int> {
+        return cars.map { car -> car.position }
+            .toList()
+    }
+
+    companion object {
+        private const val MAX_FORWARD_CONDITION = 4
+        private const val MIN_CAN_CREATE_CARS = 0
+
+        fun createCars(numberOfCar: Int): Cars {
+            require(numberOfCar > MIN_CAN_CREATE_CARS) { "0미만의 개수는 생성할 수 없습니다." }
+
+            val newCars = IntStream.range(MIN_CAN_CREATE_CARS, numberOfCar)
```

Comment:
이부분을 제가 return이 있는지모르고 repeat()를 제안드렸네요ㅠㅠ

만약 이런경우에는 [rangeTo](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.ranges/range-to.html)를 사용해서 처리하시면 될 것 같아요!!

```suggestion
            val newCars = (1..numberOfCar)
```